client/commonFramework/bindings.js

```diff
@@ -20,6 +21,12 @@ window.common = (function(global) {
     }
   };
 
+    common.ctrlShiftEFocus = function ctrlShiftEFocus() {
+        Mousetrap.bind(['command+shift+e', 'ctrl+shift+e'], () => {
+            common.editor.focus();
+        });
+    };
```

Comment:
I don't think you need to put the mousetrap bindings within a callback. They can be added without the keydown event below